src/registry_testing.jl

```diff
@@ -251,3 +251,23 @@ function test(path=pwd(); registry_deps::Vector{<:AbstractString}=String[])
     end
     return nothing
 end
+
+# Change all occurences of "digit-digit" to "digit - digit"
+function _spacify_hyphens(str::AbstractString)
+    r = r"(\d)-(\d)"
+    s = s"\1 - \2"
+    new_str = replace(str, r => s)
+end
+
+# Apply `_spacify_hyphens()` recursively through a dictionary
+function _spacify_hyphens(dict::Dict{K, V}) where {K, V}
+    new_dict = Dict{K, V}()
+    for (k, v) in pairs(dict)
```

Comment:
No need to wrap a `Dict` in `pairs` to iterate over it.

Reply:
https://github.com/JuliaRegistries/RegistryCI.jl/pull/520

Reply:
Fixed by https://github.com/JuliaRegistries/RegistryCI.jl/pull/520